### ISSUE TITLE
Default Keyboard Shortcut for toggleTerminal Does Not Match Documentation

### ISSUE THREAD

**Issue by djbreen7** [microsoft/vscode#116397]
Opening as a bug since I don't think the documentation is wrong. I think something strange happened in a recent update.

The default shortcut to toggle the intergrated terminal change from `CTRL` + `` ` `` to `CTRL` + `'`. Notice from the screenshot that the source is `Default` and `Reset Keybinding` is grayed out.  

>![Toggle Terminal](https://user-images.githubusercontent.com/13296622/107595059-d7091b80-6bd0-11eb-8101-27ded806d38f.png) 

`CTRL` + `` ` `` is now mapped to `splitEditor`:  

> ![Split Editor](https://user-images.githubusercontent.com/13296622/107595178-294a3c80-6bd1-11eb-8a66-1741bd6bdb7c.png)

- VS Code Version: 1.53.1
- OS Version: Windows 10 Enterprise 20H2

Steps to Reproduce:

1. Press `CTRL` + `` ` ``
2. Expect The Integrated Terminal to open/close

Does this issue occur when all extensions are disabled?: Yes

**Comment by meganrogge** [comment]
![snip](https://user-images.githubusercontent.com/29464607/107601974-0cb40180-6bdd-11eb-9f06-7120be330460.PNG)
It's looking correct to me and using the Ctrl and backtick works in toggling the terminal

**Comment by meganrogge** [comment]
Try Insider's perhaps?

**Comment by Tyriar** [comment]
What keyboard layout are you using?

**Comment by djbreen7** [comment]
@Tyriar English / US if you're asking about the windows keyboard layout.

No keymaps curently installed if you're asking about VS Code. I had installed IntelliJ keymaps at one point, but I've since removed them. I think VS Code should say `extentsion` rather than `default` though if the key map is the issue.

**Comment by djbreen7** [comment]
@meganrogge Same issue on insiders. Didn't install anything additional.

**Comment by Tyriar** [comment]
I can't repro here:

![image](https://user-images.githubusercontent.com/2193314/108053486-55672280-7002-11eb-91e6-2a4c591269bc.png)

Do you have en-US set as the system language as well?

**Comment by djbreen7** [comment]
@Tyriar @meganrogge I did a little digging this morning. Everything seems to be configured correctly, but there are other factors outside of VS Code that are likely contributing to my problem. I've moved this to Stack Overflow at the link below.  

[What Causes Default VS Code Keyboard Shortcuts to Differ From Official Defaults?](https://stackoverflow.com/questions/66319969/what-causes-default-vs-code-keyboard-shortcuts-to-differ-from-official-defaults)
